src/compat/eme/custom_media_keys.ts

```diff
@@ -158,7 +161,20 @@ let CustomMediaKeys : IMockMediaKeysConstructor =
     }
   };
 
-if (navigator.requestMediaKeySystemAccess) {
+/**
+ * Since Safari 12.1, EME APIs are available without webkit prefix.
+ * However, it seems that since fairplay CDM implementation within the browser is not
+ * standard with EME w3c current spec, the requestMediaKeySystemAccess API doesn't resolve
+ * positively, even if the drm (fairplay in most cases) is supported.
+ *
+ * Therefore, we prefer not to use requestMediaKeySystemAccess on Safari when webkit API
+ * is available.
+ */
+const preferWebKitAPIForSafari = isSafari &&
```

Comment:
That's a weird control flow. What if the webkit condition below gets updated? We won't know that we would break that